engine/src/test/java/org/camunda/bpm/engine/test/api/mgmt/ManagementServiceTest.java

```diff
@@ -464,16 +465,6 @@ public void testSetJobRetriesJobIdNull() {
       .hasMessageContaining("You must specify exactly one of jobId, jobIds or jobDefinitionId as parameter.");
   }
 
-  @Test
-  public void testSetJobRetriesNegativeNumberOfRetries() {
-    try {
-      managementService.setJobRetries("unexistingjob", -1);
-      fail("ProcessEngineException expected");
-    } catch (ProcessEngineException re) {
-      testRule.assertTextPresent("The number of job retries must be a non-negative Integer, but '-1' has been provided.", re.getMessage());
-    }
-  }
-
```

Comment:
I think this test case is unrelated. Why should we remove it?

Reply:
This test is a duplicate of [this test](https://github.com/camunda/camunda-bpm-platform/blob/c746c322d7b9cd3b1507d75141769a3c7164f62a/engine/src/test/java/org/camunda/bpm/engine/test/api/mgmt/ManagementServiceTest.java#L373-L379).